medusa/server/api/v2/config.py

```diff
@@ -279,6 +279,10 @@ def patch(self, *args, **kwargs):
                 theme_name = data['theme']['name']
                 app.THEME_NAME = theme_name
                 done_data['themeName'] = theme_name
+            if key == 'backlogPeriod':
+                backlog_period = data['backlogPeriod']['period']
```

Comment:
What I'm saying is PATCH is expecting something like the code above, right?

Why not send a PATCH of just this?
```
{
    backlogPeriod: 7
}
```